internal/lib/lib.go

```diff
@@ -550,10 +550,20 @@ func IsNodePortMode() bool {
 	return false
 }
 
+// ToDo: Set the Service Type only once. But this creates a problem in UTs,
+// because different types of Services needs to be tested in the UTs.
 func GetServiceType() string {
 	return os.Getenv(SERVICE_TYPE)
 }
 
+func AutoAnnotateNPLSvc() bool {
+	autoAnnotateSvc := os.Getenv(autoAnnotateService)
+	if GetServiceType() == NodePortLocal && autoAnnotateSvc != "false" {
+		return true
```

Comment:
this will return true if the user passes the value as "False".

Reply:
corrected to do case insensitive match